src/project.ts

```diff
@@ -332,6 +364,58 @@ class Project extends ServiceObject {
       }
     );
   }
+
+  getIamPolicy(): Promise<GetIamPolicyResponse>;
+  getIamPolicy(callback: GetIamPolicyCallback): void;
+  /**
+   * @typedef {array} GetIamPolicyResponse
+   * @property {Policy} 0 The new [policy]{@link https://cloud.google.com/resource-manager/reference/rest/Shared.Types/Policy}.
+   */
+  /**
+   * @callback GetIamPolicyCallback
+   * @param {?Error} err Request error, if any.
+   * @param {Policy} policy The new [policy]{@link https://cloud.google.com/resource-manager/reference/rest/Shared.Types/Policy}.
+   */
+  /**
+   * Get Iam Policy for Project.
+   *
+   * @see [projects: getIamPolicy API Documentation]{@link https://cloud.google.com/resource-manager/reference/rest/v1/projects/getIamPolicy}
```

Comment:
This API accepts options. Should we add support here?

Reply:
Done!